modules/integration_azure-mysql/conf/00-heartbeat.yaml

```diff
@@ -0,0 +1,13 @@
+module: Azure MySQL
+name: heartbeat
+
+transformation: true
+aggregation: ".mean(by=['azure_resource_id'])"
```

Comment:
I am afraid this id could change too often and cause undesirable alerts.
using same aggregation than other detectors could make this heartbeat less sensible to user change ? perhaps there is no possible change on it or you really want to be alerted of this kind of change, just a remark.
in any case using the id is not the most understandable when you receive an alert, in this case, do not use aggregation at all should provide same alerting behavior but the user will get all information instead of the id only.

Reply:
I kept the previous detector (written manually in TF) behavior which already deployed on many customers.
That's not really the purpose of this MR to challenge past detector (just wanted to rewrite using yaml conf)

Reply:
ok ok no problem